providers/amazon/src/airflow/providers/amazon/aws/log/cloudwatch_task_handler.py

```diff
@@ -222,7 +211,8 @@ def _event_to_dict(self, event: dict) -> dict:
 
     def _event_to_str(self, event: dict) -> str:
         event_dt = datetime.fromtimestamp(event["timestamp"] / 1000.0, tz=timezone.utc)
-        formatted_event_dt = event_dt.strftime("%Y-%m-%d %H:%M:%S,%f")[:-3]
+        # Format a datetime object to a string in Zulu time without milliseconds.
+        formatted_event_dt = event_dt.strftime("%Y-%m-%dT%H:%M:%SZ")
```

Comment:
Ditto: why did you change this? This seems to be unrealted to OOM behavoviour, and should have been in a different PR if it was going to be changed.

Please keep changes focused to a single thing.